data/ExperimentDesignPresets.ts

```diff
@@ -0,0 +1,26 @@
+import {EmptyAAExperiment} from "../types/experiments";
+
+interface Preset<T> {
+  name: string;
+  description: string;
+  presets: Partial<T>
+}
+
+export const AAType: Preset<EmptyAAExperiment> = {
```

Comment:
Do we have (or need) any tests to validate that this experiment matches the current schema; in case the schema changes and this falls out of sync? Or will TypeScript fail automatically if a new field is added to the schema and this doesn't conform?

Reply:
In this case because `EmptyAAExperiment` extends `Experiment`, typescript will fail to compile with the following message (for example, if an extra `foo` property were added)
```
Object literal may only specify known properties,
and 'foo' does not exist in type 'Preset<EmptyAAExperiment>'.
```

...although i do need to add `data` to the tsconfig

Reply:
The `Partial` on `Preset` does allow for properties to be *missing* though, which is what we want for presets.

Reply:
> ...although i do need to add `data` to the tsconfig

Silly question, but is your "data/" directory here the same as @mythmon's newly added "data/" directory in his draft #43 PR? (or is it different concepts/objectives but name collision?)

Reply:
I believe that they are roughly the same goal, but Kate and I haven't coordinated to make sure that both implementations work well together (and in fact they won't if both PRs merge).